src/aws_encryption_sdk/internal/formatting/encryption_context.py

```diff
@@ -47,9 +47,15 @@ def assemble_content_aad(message_id, aad_content_string, seq_num, length):
     return struct.pack(fmt, message_id, aad_content_string.value, seq_num, length)
 
 
-# help function to simplify code
-# this is necessary to pass flake8
+def _key_or_value_too_large(encryption_context_list):
+    """Helper method to check whether a key or value are too long."""
+    longest_key = max(encryption_context_list, key=lambda x: len(x[0]))[0]
+    longest_value = max(encryption_context_list, key=lambda x: len(x[1]))[1]
+    return len(longest_key) > 2 ** 16 - 1 or len(longest_value) > 2 ** 16 - 1
+
+
 def _serialize_encryption_context_list(encryption_context_list, serialized_context):
+    """Helper function to serialize the list of encryption context key-value pairs."""
```

Comment:
Sorry, I offset some lines a bit when I was making that comment and picked the wrong example.

Reply:
yup flake8 enforced this